.appveyor.yml

```diff
@@ -18,12 +18,11 @@ install:
   - set PATH=%CONDA_INSTALL_LOCATION%;%CONDA_INSTALL_LOCATION%\scripts;%PATH%;
   - conda config --set always_yes true  # else AppVeyor will hang forever waiting for user ([y]/n)
   - conda update -n base conda
-  - conda env create -q -f environment.yml python=3.6
+  - conda env create -q -f environment.yml
   - activate qcodes
   - pip install -r test_requirements.txt
   - pip install -r docs_requirements.txt
-  - python -m pip install --upgrade pip
-  - pip install .
+  - pip install -e .
```

Comment:
`-e` for appveyor and no `-e` for Travis? why not `-e` for both?

Reply:
We want to test both types of install. As you may remember there are plenty of examples of regular install being broken and editable working. (Because editable just puts in a link and does not care about the install manifest)  But most scientist (at least in Copenhagen) uses an editable on windows so imho it makes the most sense to test these together

Reply:
ok, agree.